Content.Shared/AlertLevel/ChangeAlertLevel.cs

```diff
@@ -1,8 +1,17 @@
+using Robust.Shared.Audio;
 using Robust.Shared.Serialization;
 
 namespace Content.Shared.ChangeAlertLevel
 {
-    public abstract partial class SharedChangeAlertLevelComponent : Component
+    public sealed partial class SharedChangeAlertLevelComponent : Component
     {
+        [ViewVariables(VVAccess.ReadWrite), DataField(required: true)]
```

Comment:
remove `ViewVariables(VVAccess.ReadWrite), ` for all these its automatically readwrite

Reply:
👍